git-refresher/pull-requests.qmd

```diff
@@ -0,0 +1,99 @@
+--- 
```

Comment:
Let's shift this into the GitHub refresher section and add a line at the start to say "You'll need two branches to merge to be able to create a PR. If you haven't already, follow the steps [create a branch], [making changes] and [create more branches] to be able to create and complete a mock pull request."